includes/class-wc-stripe-order-helper.php

```
@@ -891,6 +901,27 @@ public function update_stripe_upe_redirect_processed( ?WC_Order $order = null, b
 		return $this->update_order_meta( $order, self::META_STRIPE_UPE_REDIRECT_PROCESSED, $redirect_processed );
 	}
 
+	/**
+	 * Gets the In-Person Payments channel for the order.
+	 *
+	 * @param WC_Order|null $order
+	 * @return false|string|null
+	 */
+	public function get_stripe_ipp_channel( ?WC_Order $order = null ) {
+		return $this->get_order_meta( $order, self::META_STRIPE_IPP_CHANNEL );
+	}
```

PHPMD (2.15.0)

[error] The method get_stripe_ipp_channel is not named in camelCase. (undefined)

(CamelCaseMethodName)